src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -137,7 +139,10 @@ private void driveWrist(TrapezoidProfile.State state) {
     public void setArmTarget(double targetPos, double targetVel) {
         targetPos = getArmClampedGoal(targetPos);
 
-        if(positionForbidden(targetPos, getWristPos())) return;
+        if(positionForbidden(targetPos, getWristPos())) 
+        
+
+        return;
```

Comment:
Is something supposed to go here? Don't add unnecessary spaces (Alt+Shift+F will auto format your code). Also if you think some code needs to be written here for manipulator rumbling, what about wrist?